src/javascripts/base_app.js

```diff
@@ -95,6 +97,8 @@ BaseApp.prototype = {
 
   switchTo: function(name, data) {
     this.$('[data-main]').html(this.renderTemplate(name, data));
+    var newHeight = Math.min($('html').height(), maxHeight);
+    this.zafClient.invoke('resize', { height: newHeight, width: '100%' });
```

Comment:
That's an unusual case and does not mirror the behaviour of legacy apps where the container div will always expand.
If that behaviour is needed developers should create a container element in their template and give it an appropriate `overflow` style using css.